app/views/teaching_events/index.html.erb

```diff
@@ -5,27 +5,6 @@
     <p>
       Find out more about getting into teaching at a free event where you can get all your questions answered.
     </p>
-
-    <details class="teaching-events__descriptions">
-      <summary class="teaching-events__descriptions--summary">Explore the different types of events</summary>
-
-      <dl>
-        <div>
-          <dt>Get Into Teaching events</dt>
-          <dd>Get expert advice, talk to teachers and connect with people like you considering a career in teaching. Run by the Department for Education (DfE).</dd>
```

Comment:
@philspray If we are adding back in the GIT event description, can we please add 'meet training providers' as well, as this is a really key part of it